lib/core/src/Cardano/Wallet.hs

```diff
@@ -357,13 +360,15 @@ newWalletLayer
     :: forall s t. (Buildable (Tx t))
     => Trace IO Text
     -> Block (Tx t)
-        -- ^ Very first block
+        -- ^ Very first block (block0)
+    -> UTCTime
```

Comment:
See also #527 because this is also adding config parameters to the wallet layer.